src/components/Link/index.tsx

```diff
@@ -35,8 +39,7 @@ export const Link = ({ children, to, props }: LinkProps): JSX.Element => {
         ...history.state.contentState,
         {
           id,
-          name: linkItem.metadata.name,
-          props
```

Comment:
I think we need to keep `props` here...
This is for when we with back navigation need to recreate a previous, now unmounted, component.

Reply:
It's possible that we need to revisit this later on, right now it _might_ work without, but when we need to persist state for unmounted views, I think this is the place to do it with the current solution.

Reply:
I'll leave it for later then.